platform/security/sts/security-sts-server/src/main/resources/OSGI-INF/blueprint/cxf-sts.xml

```diff
@@ -172,9 +172,7 @@
     </bean>
 
     <bean id="service"
-          class="ddf.security.sts.StsStaticService">
-        <cm:managed-properties persistent-id="ddf.security.sts.StsStaticService"
```

Comment:
Oh yeah good catch. I was still testing the removal of the `StsStaticService`. I will follow-up with removing the Java class and all references. It was originally used as a workaround for https://issues.apache.org/jira/browse/CXF-6673